index.js

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env ./node_modules/.bin/babel-node
+
+
+import nomnom from 'nomnom';
+import glob from 'glob';
+import path from 'path';
+
+
+// Find all available scaffolds
```

Comment:
In the future, we should scan for all globally installed scaffolds, but that should be a different PR. =)